device_storage.py

```diff
@@ -1,4 +1,5 @@
 import Domoticz
+import sys
```

Comment:
Thank you for adding thermostat support to the plugin and may I ask you to revert changes in this file as they don't seem required

Reply:
debug changes in device_storage.py are reverted now